INITIAL_SETUP.md

```diff
@@ -0,0 +1,203 @@
+    data storage. Once the project is created, click on "Database" and "Create
+    database".
+
+1.  Follow instructions to
+[modify/set up firestore rules](#modifyset-up-firestore-rules).
+
+1.  Go to the Firestore database. To get there, click on "Database" in the left-
+    hand sidebar.
+    [Here](https://firebase.corp.google.com/project/mapping-crisis/database/firestore/data~2F)
+    is the current page.
+
+1.  Create a collection called `ALLOWED_USERS`. In that collection, create a
+    document called `ALL_USERS`. In that document, create a field of array type
+    called `USERS`. To that array, add the email addresses of users who should
+    be able to view/edit user-defined features on the map. The current data
+    looks like: ![allowed users screenshot](./markdown_images/allowed_users.png)
+
+1.  Go to [Credentials](https://console.developers.google.com/apis/credentials),
+    or "Credentials" under https://console.developers.google.com/ if that link
+    does not work.
+
+1.  Setting up an API Key.
+
+    -  You should see an "API Keys" section, and under that "Browser key (auto
+       created by Firebase)", or similar:
+       ![credentials browser key screenshot](./markdown_images/credentials_browser_key.png)
+       Click on it.
+
+    -  Rename the key so it's easier to track (for instance, the current key is
+       called "Mapping browser key") and set API restrictions. Choose the
+       following five APIs:
+           -   Maps JavaScript API
+           -   Google Earth Engine API
+           -   Identity Toolkit API
+           -   Places API
+           -   Token Service API (only needed for optional token server)
+  
+       ![key apis screenshot](./markdown_images/key_apis.png)
+  
+       Remember to save your changes!
+
+    -  Copy the API key to the `API_KEY` constant in 
+       [common_auth_utils.js](./docs/common_auth_utils.js). Also copy it at the
+       end of the `script` tag for the Maps API in
+       [index.html](./docs/index.html),
+       [manage_disaster.html](./docs/import/manage_disaster.html), and
+       [script_loader.js](./cypress/support/script_loader.js).
+
+1.  Setting up OAuth client ID.
+
+    -  Go back to the
+       [Credentials](https://console.developers.google.com/apis/credentials)
+       page. Below the "API Keys" section is an OAuth 2.0 Client IDs section. It
+       should have an auto-generated one. Click on it.
+
+    -  Give it a useful name (the current one is "Disaster relief mapping").
+  
+    -  Specify "Authorized JavaScript origins". Currently they are
+       `http://localhost:8080` for local development and
+       `https://givedirectly.github.io`. This second origin should be the domain
+       that the mapping page is hosted on.
+  
+    -  Specify "Authorized redirect URIs". You must list here every page that a
+       normal user or the Admin user might go to in the app, including when
+       managing a disaster. Currently, it is `http://localhost:8080`,
```

Comment:
yes, if it's usable. Although the image collection page is kind of its own beast in terms of the authenticator (since it needs to use GCS) so we'll have to see how it gets made usable.